### ISSUE TITLE
it's hard to training

### ISSUE THREAD

**Issue by cena001plus** [zhixuhao/unet#126]
to simple two class,  i found it's very diffcult to training ,the result always  different for training each time.

**Comment by tuanphan09** [comment]
I used this repo's model to train with my own data. It's unstable, sometime I got good result but most of the time the training stucked in local optima too.
My solution to fix this:
- Decrease learning rate
- Remove all dropout layer + reduce number of filters in all Conv2D by half.
